app/connectors/VatObligationsConnector.scala

```diff
@@ -77,4 +77,31 @@ class VatObligationsConnector @Inject()(http: HttpClient,
         httpError
     }
   }
+
+  def getVatReturnObligations(vrn: String, status: Status.Value)
```

Comment:
Can't this method just have optinal from and to parameters? Instead of creating a repeated method with different params? It'll save a lot of code since only one line in the methods are different